### ISSUE TITLE
[Config Support]:  Request to Maintain Bear Label Support

### ISSUE THREAD

**Issue by zettalion** [blakeblackshear/frigate#8696]
### Describe the problem you are having

I've been using Frigate for years - and in my part of the country, bears are a real issue.   I've integrated bear detection and notification into an Alexa voice broadcast and phone notifications so my my kids don't stumble across bears near our house.  (Especially on trash day, because the bears have learned the day of the week that folks will be rolling out their bear resistant trash bins).

I completely understand that a subset of labels needed to be started with in Frigate+,  I'm just respectfully asking the team to consider adding bear labels,  as I will commit to providing years of training data for the bears on my property, including various seasons and camera angles.

For me,  I have all the other animals also in my camera feeds - dogs, cats, and deer --> however none pose the safety issue that bears do.   Especially in spring when they come out of hibernation.  

In summary,  I'd like to continue to support the Frigate+ project - and will do the heavy lifting in providing Bear data (As obviously not many communities have this problem--> even with the rarity,  Bear is the absolute top priority for me and my family.

Thanks a bunch for your consideration!

### Version

0.12.1-367D724

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### Frigate stats

```json
N/A
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

USB

**Comment by NickM-27** [comment]
As has been discussed at https://github.com/blakeblackshear/frigate/discussions/7932 more labels will be coming in the future

**Comment by zettalion** [comment]
Thanks for responding.   I'm just hoping there is a way we can help contribute by rank ordering the value of each label.  ie., each attribute could be tagged with root justification of including said label.  
a) value justification (ie., loss avoidance, safety, etc.)
b) scope of applicability (how many folks would stand to benefit from the new label).

For example,  including packages makes sense in Frigate plus -- and if we break down why we find that valuable, you could start to break it out in the way I began to highlight above... 
a) value  justification could be written like "Theft Avoidance".    So that could be ranked say 8/10 in value from that metric.  And you could perhaps multiply that ranking by the scope of applicability ie.,  number of people that are expected to benefit -- you could do it by state.   So if all 50 states would benefit, you would give it full marks.  100% of states means 10/10 on that metric.  

For my 'bear label' request,  Safety would be the justification,  so it would hopefully get 10/10 on that ranking,  however admittedly because only 20% of the states have a concern with bears,  it would only be 2/10 on the "scope" metric.

I read all of the comments on adding future labels - I would be happy to contribute some kind of quantitative method for your team to triage which labels are added at a future date if that helps.

**Comment by ehn** [comment]
> So if all 50 states would benefit, you would give it full marks. 100% of states means 10/10 on that metric.

Frigate, like most open-source software, is used outside of the US.

**Comment by blakeblackshear** [comment]
When I add the next set of labels, I plan to make sure the effort required to add more will be low. I don't think we will need a complex framework for prioritizing new labels.
